shell.nix

```diff
@@ -135,6 +142,7 @@ let
       cardano-node.cardano-cli
       hsPkgs.hydra-node.components.exes.hydra-node
       hsPkgs.hydra-tui.components.exes.hydra-tui
+      run-hydra-demo
```

Comment:
Can you please add something to the user manual about this?

Reply:
Yeah, of course.